lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -52,6 +52,11 @@ def capture(amount, authorization, options={})
         add_transaction_elements(post, 'CAPTURE', options)
         add_reference(post, authorization)
 
+        if !amount.nil? && amount.to_f != 0.0
+          post[:transaction][:additionalValues] ||= {}
+          post[:transaction][:additionalValues][:TX_VALUE] = invoice_for(amount, options)[:TX_VALUE]
```

Comment:
@curiousepic What extra values? We only include just the `TX_VALUE` key here, which matches the Zendesk ticket.

Reply:
I see now, kind of ignored that `[:TX_VALUE]` on the end there. They do also include currency in the example (despite saying it's ignored), but we can go with this for now and address any further issues as they come up.

Reply:
Yeah, there's some disagreement there; I'm having to take them at face value on their example, ’cause that's all I've got to go with.